language/transaction-builder/generator/README.md

```diff
@@ -195,6 +197,58 @@ dotnet add Demo.csproj reference ../Bcs/Bcs.csproj
 dotnet run --project Demo.csproj
 ```
 
+### Swift
+
+To install Swift source `Serde`, `Bcs`, `DiemTypes`, and `DiemStdlib` into a target directory `$DEST`, run:
+```bash
```

Comment:
I suspect the e2e test in CI currently assumes that every language has two blocks of bash.